src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json

```diff
@@ -0,0 +1,62 @@
+    "System.Collections.Concurrent": "4.0.10-beta-22819",
+    "System.Collections.NonGeneric": "4.0.0-beta-22819",
+    "System.Collections.Specialized": "4.0.0-beta-22819",
+    "System.ComponentModel": "4.0.0-beta-22819",
+    "System.ComponentModel.EventBasedAsync": "4.0.10-beta-22819",
+    "System.Diagnostics.Contracts": "4.0.0-beta-22819",
+    "System.Diagnostics.Debug": "4.0.10-beta-22819",
+    "System.Diagnostics.Tools": "4.0.0-beta-22819",
+    "System.Globalization": "4.0.10-beta-22819",
+    "System.IO": "4.0.10-beta-22819",
+    "System.IO.Compression": "4.0.0-beta-22819",
+    "System.Linq": "4.0.0-beta-22819",
+    "System.Linq.Expressions": "4.0.10-beta-22819",
+    "System.Linq.Queryable": "4.0.0-beta-22819",
+    "System.Net.Http": "4.0.0-beta-22819",
+    "System.Net.NameResolution": "4.0.0-beta-22819",
+    "System.Net.Primitives": "4.0.10-beta-22819",
+    "System.Net.Security": "4.0.0-beta-22819",
+    "System.Net.Sockets": "4.0.0-beta-22819",
+    "System.Net.WebHeaderCollection": "4.0.0-beta-22819",
+    "System.ObjectModel": "4.0.10-beta-22819",
+    "System.Reflection": "4.0.10-beta-22819",
+    "System.Reflection.DispatchProxy": "4.0.0-beta-22819",
+    "System.Reflection.Emit": "4.0.0-beta-22819",
+    "System.Reflection.Extensions": "4.0.0-beta-22819",
+    "System.Reflection.Primitives": "4.0.0-beta-22819",
+    "System.Reflection.TypeExtensions": "4.0.0-beta-22819",
+    "System.Resources.ResourceManager": "4.0.0-beta-22819",
+    "System.Runtime": "4.0.20-beta-22819",
+    "System.Runtime.Extensions": "4.0.10-beta-22819",
+    "System.Runtime.Handles": "4.0.0-beta-22819",
+    "System.Runtime.InteropServices": "4.0.20-beta-22819",
+    "System.Runtime.Serialization.Xml": "4.0.10-beta-22819",
+    "System.Security.Claims": "4.0.0-beta-22819",
+    "System.Security.Cryptography.Encoding": "4.0.0-beta-22819",
+    "System.Security.Cryptography.Encryption": "4.0.0-beta-22819",
+    "System.Security.Cryptography.Hashing": "4.0.0-beta-22819",
+    "System.Security.Cryptography.RandomNumberGenerator": "4.0.0-beta-22819",
+    "System.Security.Cryptography.X509Certificates": "4.0.0-beta-22819",
+    "System.Security.Principal": "4.0.0-beta-22819",
+    "System.Security.Principal.Windows": "4.0.0-beta-22819",
+    "System.Text.Encoding": "4.0.10-beta-22819",
+    "System.Threading": "4.0.10-beta-22819",
+    "System.Threading.Tasks": "4.0.10-beta-22819",
+    "System.Threading.Timer": "4.0.0-beta-22819",
+    "System.Xml.ReaderWriter": "4.0.10-beta-22819",
+    "System.Xml.XDocument": "4.0.10-beta-22819",
+    "System.Xml.XmlDocument": "4.0.0-beta-22819",
+    "System.Xml.XmlSerializer": "4.0.10-beta-22819",
+    "xunit": "2.0.0-beta5-build2785",
+    "xunit.abstractions.netcore": "1.0.0-prerelease",
+    "xunit.assert": "2.0.0-beta5-build2785",
+    "xunit.core.netcore": "1.0.1-prerelease",
+    "xunit.netcore.extensions":  "1.0.0-prerelease-*"
+  },
+  "frameworks": {
+    "dnxcore50": {}
+  }
+}
```

Comment:
Nit: GitHub will show this icon whenever the file is missing a final newline, so ultimately we need a newline here (and anywhere else this icon shows).